code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -260,14 +260,19 @@
 	S.DisIntegrate(src)
 	return TRUE
 
-/obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+/obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S, var/datum/gas_mixture/air_contents)
 	var/isonshuttle = istype(get_area(src), /area/shuttle)
 	for(var/turf/T in range(1, src))
+		var/datum/gas_mixture/pressure_check = T.return_air()
 		var/area/A = get_area(T)
 		if(isspaceturf(T) || (!isonshuttle && (istype(A, /area/shuttle) || istype(A, /area/space))) || (isonshuttle && !istype(A, /area/shuttle)))
 			to_chat(S, "<span class='warning'>Destroying this object has the potential to cause a hull breach. Aborting.</span>")
 			S.target = null
-			return FALSE
+			return TRUE
```

Comment:
Done here to fix AI swarmers not being able to destroy doors once another swarmer hit a wall next to space or such.
Please note this in the PR description and CL

Reply:
I'm not sure what the change here is, since it's already set to `return TRUE`?

Reply:
You've changed it from `FALSE` to `TRUE`. Which is correct but you've not mentioned this in the CL and description of your PR. It's a fix since now AI swarmers will stop deconstructing doors once another AI swarmer tries to decon a door adjacent to space or such.

Reply:
I think I'm misreading something here, doesn't `return TRUE` mean that AI swarmers will destroy whatever it is?
Which would mean that the `return TRUE` in the SM containment would be a bad thing.

This is what I'm looking at:
> //return TRUE/FALSE whether or not an AI swarmer should try this swarmer_act() again, NOT whether it succeeded.

Reply:
It will use that value to check if it should try any objects of the same type again. If you make it return FALSE here it will never try to eat doors again. Some for the walls. It will add the type of the object to a blacklist